examples/check_bt.sh

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+export BT=$(head -1 ../deployment/bigtable.txt)
+export PROJECT=google.com:datcom-store-dev
+echo $BT
+echo $PROJECT
+
+# Check get_stats cache
+echo "Get stats for d/f/country/AUT^count^measured^^^^Person^gender^Male"
+cbt -project=$PROJECT -instance prophet-cache read ${BT} prefix="d/f/country/AUT^count^measured^^^^Person^gender^Male"
+echo "Get stats for d/f/country/AUT^count^measured^Count_Person_25To64Years^^^Person^age^Years25To64^educationalAttainment^LessThanPrimaryEducation&PrimaryEducation&LowerSecondaryEducation"
+cbt -project=$PROJECT -instance prophet-cache read ${BT} prefix="d/f/country/AUT^count^measured^Count_Person_25To64Years^^^Person^age^Years25To64^educationalAttainment^LessThanPrimaryEducation&PrimaryEducation&LowerSecondaryEducation"
```

Comment:
This looks useful for debugging, should we make this an argument which defaults to this prefix?

Reply:
I wan to throw a bunch of different bt cache keys in here for all cache types, that might need a lot arguments? what would be a good bash for this purpose?

Reply:
ok, made the prefix as key and move the concrete ones as list comments above

Reply:
What you have works. Alternatively, we can accept input on stdin (refer: https://screenshot.googleplex.com/mhkM9hFpCDs), so they can do like below.

# echo "<KEY1>" | ./check_bt.sh

# cat key_file.txt
<KEY1>
<KEY2>
...
# cat key_file.txt | ./check_bt.sh